docs/developing_iree/profiling_cpu_events.md

```diff
@@ -0,0 +1,13 @@
+# CPU cache and other CPU event profiling
+
+## Android
+
+On Android we can use `simpleperf`. It's preinstalled on current Android `userdebug` images, and part of the Android NDK.
+
+TODO write me
+
+## Linux
+
+On desktop Linux we can use `perf`.
+
+TODO write me
```

Comment:
Link to https://perf.wiki.kernel.org/index.php/Main_Page for now?

Reply:
done!